test/test_cli.rb

```diff
@@ -107,6 +106,7 @@ def test_control_clustered
 
     cli.launcher.stop
     t.join
+    s.close
```

Comment:
Hmmmm... why is this necessary? Shouldn't Puma do this by itself?

Reply:
As mentioned, I can't just play around with Unix Sockets locally.  But:

1. The test creates `s`, not Puma. Hence, the test should close it.  Think 'not quite right' PR's.
2. `teardown` doesn't have access to `s`., but it should unlink the file if Puma doesn't.

I'll remove if you'd like.  I was just trying to make the tests bomb-proof.  If we need more asserts or tests, that's another thing...